proof/invariant-abstract/ARM/ArchAcc_AI.thy

```diff
@@ -2543,38 +2543,59 @@ lemma create_mapping_entries_valid_slots [wp]:
   apply (fastforce intro!: aligned_add_aligned is_aligned_shiftl_self)
   done
 
-lemma is_aligned_addrFromPPtr_n:
-  "\<lbrakk> is_aligned p n; n \<le> 28 \<rbrakk> \<Longrightarrow> is_aligned (Platform.ARM.addrFromPPtr p) n"
-  apply (simp add: Platform.ARM.addrFromPPtr_def)
-  apply (erule aligned_sub_aligned, simp_all)
-  apply (simp add: pptrBaseOffset_def Kernel_Config.physBase_def
-                   pptrBase_def pageBits_def)
-  apply (erule is_aligned_weaken[rotated])
-  apply (simp add: is_aligned_def)
-  done
+(* FIXME MK: staging area for physBase abstraction *)
+
+(* FIXME MK: candidate interface lemma  *)
+lemma physBase_aligned[simplified pageBitsForSize_simps]:
+  "is_aligned physBase (pageBitsForSize ARMSuperSection)"
+  by (simp add: is_aligned_def Kernel_Config.physBase_def)
```

Comment:
This one needs to go into Kernel_Config_Lemmas, but where should the ones below go? ArchInvariants? Leave them here? ArchAcc is a strange file, I'm not sure of its origins.

Reply:
(recording offline discussion: should go into ArchInvariants for now)